src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -18,7 +19,7 @@
     public static final String MESSAGE_SUCCESS = "Your timetable has been cleared!";
 
     @Override
-    public CommandResult execute(Model model, CommandHistory history) {
+    public CommandResult execute(Model model, CommandHistory history) throws SecurityAuthenticationException {
```

Comment:
If it's the parser that throws the security exception, why does this throw an exception too?

Reply:
Cause for this commands that do not have parsers, this is the place to put